includes/rest/class-server.php

```diff
@@ -181,4 +183,32 @@ public static function validate_activitypub_requests( $response, $handler, $requ
 
 		return $response;
 	}
+
+	/**
+	 * Modify the parameter priority order for a REST API request.
+	 *
+	 * @param string[]        $order   Array of types to check, in order of priority.
+	 * @param WP_REST_Request $request The request object.
+	 *
+	 * @return string[] The modified order of types to check.
+	 */
+	public static function request_parameter_order( $order, $request ) {
+		$route = $request->get_route();
+
+		// Check if it is an activitypub request and exclude webfinger and nodeinfo endpoints.
+		if ( ! \str_starts_with( $route, '/' . ACTIVITYPUB_REST_NAMESPACE ) ) {
```

Comment:
`str_starts_with` was introduced in PHP 8, so we should update the min required PHP version for the plugin or replace it with `strpos`

Reply:
we use a backport for that: https://github.com/Automattic/wordpress-activitypub/blob/d532f8be06f0e3539f30ba1500dab95d391a99c6/includes/compat.php#L8